### PR TITLE
Labelmerge

### PR DESCRIPTION
**Pull request by kaitj**
## Proposed changes

Update (Nov 30): Changed the base branch to `dry-run` as the tests were helpful to making sure that the changes don't break the workflow.

* Updates `zona_bb_subcortex.smk` with rule for labelmerge
  * Adds container for labelmerge within the params, so that `singularity run` can be called. Otherwise snakemake just tries to run the rule using the container as a dependency.
* Also adds / updates resource files necessary for workflow to run
  * Note, only MNI152NLin6Asym T1w template from templateflow is added. Can easily go between the 2 commonly used MNI spaces and this is sufficient for getting a transformation to subject space. In the future, we can look at requiring fmriprep / adding an fmriprep as suggested during the weekly meeting.

One note is that this PR assumes that https://github.com/khanlab/labelmerge/pull/22 is good to proceed.

## Types of changes

What types of changes does your code introduce? Put an `x` in the boxes that apply

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionalitiy)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (if none of the other choices apply)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you are unsure about any of the choices, don't hesitate to ask!

- [ ] Changes have been tested to ensure that fix is effective or that a feature works.
- [x] Changes pass the unit tests
- [x] Code has been run through the `poe quality` task
- [x] I have included necessary documentation or comments (as necessary)
- [ ] Any dependent changes have been merged and published

## Notes
All PRs will undergo the unit testing before being reviewed. You may be requested to explain or make additional changes before the PR is accepted.

### CODE REVIEW

File: dbsc/workflow/rules/mrtpipelines.smk [hunk 1]
```diff
@@ -140,6 +140,7 @@ rule dwi2response:
 
 rule responsemean:
     """Compute average response function"""
+    # DOUBLE CHECK
```
Comment (tkkuehn) [inline dbsc/workflow/rules/mrtpipelines.smk:143]: What are we double checking here?

**Review by tkkuehn — APPROVED** [review]
Looks good, just wanna make sure the double check comment isn't blocking before I merge this.

File: dbsc/workflow/rules/mrtpipelines.smk (hunk 1 above)
Comment (kaitj) [reply dbsc/workflow/rules/mrtpipelines.smk:143]: Ahh, this was a note to my self, I removed it in one of the other PRs. Just wanted to check the conditional was written correctly by comparing with an existing rule written in another workflow.

**Comment by tkkuehn** [comment]
Tests are failing because the `labelmerge` branch doesn't have the test data. I'll merge this and make sure the tests complete on the `dry-run` branch.

**Pull request merged**